app/models/asset.rb

```diff
@@ -242,12 +242,11 @@ def self.extract_asset_text(asset_id)
         file_path = fa.path
       end
 
-      unless Yomu.class_eval('@@server_pid')
-        Yomu.server(:text, nil)
-        sleep(5)
+      if !ENV['NO_TIKA_SERVER'] && Yomu.class_variable_get(:@@server_pid).nil?
```

Comment:
Add comment `Start Tika as a server`

Reply:
So `NO_TIKA_SERVER` will need to be set for every sciNote instance, based on the fact if server has Tika installed or not. Will this be done in scope of this Jira task or as a separate one? Just mentioning it so that we won't forget it.

Reply:
there is no separate server package, it is always just one jar file which can be used in different ways. CF template is already updated